libs/internal/include/launchdarkly/data_model/segment.hpp

```diff
@@ -0,0 +1,69 @@
+#pragma once
+
+#include <boost/json/value.hpp>
+#include <launchdarkly/attribute_reference.hpp>
+#include <launchdarkly/value.hpp>
+#include <optional>
+#include <string>
+#include <tl/expected.hpp>
+#include <unordered_map>
+#include <vector>
+
+namespace launchdarkly::data_model {
+
+struct Segment {
```

Comment:
I think some blank lines between struct definitions would improve the readability.